Framework/PythonInterface/mantid/api/src/Exports/IFunction.cpp

```diff
@@ -86,6 +91,13 @@ GNU_DIAG_ON("conversion")
 GNU_DIAG_ON("unused-local-typedef")
 ///@endcond
 
+Mantid::API::Jacobian *getFunctionDeriv(IFunction &self, const Mantid::API::FunctionDomain &domain) {
```

Comment:
Could this not be?

```
Mantid::API::Jacobian *getFunctionDeriv(IFunction &self, const Mantid::API::FunctionDomain &domain) {
  auto *out = new Mantid::CurveFitting::Jacobian(domain.size(), self.nParams());
  self.functionDeriv(domain, *out);
  return out;
}
```
Maybe I've missed something, but not seeing why we need to copy construct a new one at the end.